docs/source/user-guide/getting-started.rst

```diff
@@ -10,14 +10,29 @@ This guide to getting started with conda goes over the basics of starting up and
 
    Anaconda Navigator is a graphical desktop application that enables you to use conda without having to run commands at the command line.
 
-   See `Getting started with Anaconda Navigator <https://docs.anaconda.com/free/navigator/getting-started>`_ to learn more.
+   See `Getting started with Anaconda Navigator <https://docs.anaconda.com/navigator/getting-started/>`_ to learn more.
 
 Before you start
 ================
 
-To bootstrap a conda installation, use a minimal installer such as `Miniconda <https://docs.anaconda.com/free/miniconda/>`_ or `Miniforge <https://conda-forge.org/download>`_.
+To bootstrap a conda installation, use a minimal installer such as `Miniconda <https://docs.anaconda.com/miniconda/>`_ or `Miniforge <https://conda-forge.org/download>`_.
 
-Conda is also included in the `Anaconda Distribution <https://docs.anaconda.com/free/anaconda/install/>`_.
+Conda is also included in the `Anaconda Distribution <https://docs.anaconda.com/anaconda/install/>`_.
```

Comment:
```suggestion
``conda`` is also included in the `Anaconda Distribution <https://docs.anaconda.com/anaconda/install/>`__.
```

Reply:
In this case, it's referring to the conda project, not the `conda` command, [so no need to put it in code formatting](https://github.com/conda/ceps?tab=readme-ov-file#conda-capitalization-standards). That's why it's correct to capitalize it, too.